### ISSUE TITLE
Ingest and archive CDC hospitalization data

### ISSUE THREAD

**Issue by capnrefsmmat** [cmu-delphi/covidcast-indicators#45]
From @ryantibs:

> We should start ingesting the CDC [NSHN Hospitalization Data](https://www.cdc.gov/nhsn/covid19/report-overview.html), and the CDC [COVID-NET hospitalization data](https://www.cdc.gov/coronavirus/2019-ncov/covid-data/covid-net/purpose-methods.html).  These appear to be two alternative CDC run surveillance system for state-level COVID hospitalizations.  We should compare these against each other---in whatever ways are possible---and also to state level COVID hospitalization data collected from the state depts of health, through [COVID Tracking](https://covidtracking.com/data).

From @RoniRos:

> It's important to start this ASAP, because there is backfill (aka data revisioning), and no guarantee that anyone is storing the historical versions, which are critical for real-time forecasting.

Until we download and compare these, it won't be clear what exactly should go into the API, so we should

- [ ] download the data regularly, archiving historical versions so we understand the backfill
- [ ] compare the data from the different sources
- [ ] determine which sources we want to use
- [ ] determine if these should be in the API
- [ ] put them in the API

This issue will just be for the first three steps; once we know what should go into the API and how, the Indicators team can plan what release this should go into.

**Comment by undefx** [comment]
Any status changes since the May 28 update?

Is the data still being manually downloaded to someone's computer?

@eujing @krivard

**Comment by dfarrow0** [comment]
Any status changes since the May 28 update?

Is the data still being manually downloaded to someone's computer?

@eujing @krivard

**Comment by eujing** [comment]
@dfarrow0 I think the CDC COVID-NET data is being scraped and run as a (non-public?) indicator already as `cdc_covidnet`

**Comment by dfarrow0** [comment]
thanks @eujing !

@krivard you might want to close this issue out given https://github.com/cmu-delphi/covidcast-indicators/pull/79

**Comment by krivard** [comment]
No, COVID-NET is not being run. It's been merged to main, but has no `deploy-*` branch and has not been automated (or if it has, nobody told me...)

iirc the coverage was not high enough to deploy it for states, and [we don't yet support HHS regions](https://github.com/cmu-delphi/delphi-epidata/issues/192).
